### PR TITLE
vendor: Switch from godep to glide tool

### LINKED ISSUE #82: vendor: Switch from Godeps tool to glide
* Remove the Godeps directory
* https://github.com/kardianos/govendor

### PR DESCRIPTION
**Pull request by dghubble**
* Use glide for dependency management
* Vendored an old version of camlistore to please Ignition v1 
* This simplifies working on some dependant projects and closes #82
* Discussion https://github.com/coreos/docs/issues/775

cc @jonboulle @joeatwork

### CODE REVIEW

File: Documentation/dev/develop.md [hunk 1]
```diff
@@ -41,4 +41,19 @@ Run the ACI with rkt on `metal0`.
 
 Alternately, run the Docker image on `docker0`.
 
-    sudo docker run -p 8080:8080 --rm -v $PWD/examples:/var/lib/bootcfg:Z -v $PWD/examples/groups/etcd-docker:/var/lib/bootcfg/groups:Z coreos/bootcfg:latest -address=0.0.0.0:8080 -log-level=debug
\ No newline at end of file
+    sudo docker run -p 8080:8080 --rm -v $PWD/examples:/var/lib/bootcfg:Z -v $PWD/examples/groups/etcd-docker:/var/lib/bootcfg/groups:Z coreos/bootcfg:latest -address=0.0.0.0:8080 -log-level=debug
+
+## Dependencies
+
+Project dependencies are commited to the `vendor` directory, so Go 1.6+ users can clone to their `GOPATH` and build or test immediately.
```
Comment (ericchiang) [inline Documentation/dev/develop.md]: May want to comment that Go 1.5 users can set `GO15VENDOREXPERIMENT=1` in their bash profile.

File: Documentation/dev/develop.md [hunk 2]
```diff
@@ -41,4 +41,19 @@ Run the ACI with rkt on `metal0`.
...
+
+Project developers should use [glide](https://github.com/Masterminds/glide) to manage commited dependencies under `vendor`. Configure `glide.yaml` as desired. Use `glide update` to download and update dependencies listed in `glide.yaml` into `/vendor`.
```
Comment (ericchiang) [inline Documentation/dev/develop.md]: I might be explicit to NEVER use glide get. But maybe that's something that should go in coreos/docs.

Comment (dghubble) [inline Documentation/dev/develop.md]: added a tiny blurb

File: Documentation/dev/develop.md (hunk 1 above)
Comment (dghubble) [inline Documentation/dev/develop.md]: added

**Pull request merged**